### ISSUE TITLE
[FE] Auth Profile - Address Change Messaging - Build FE

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#33774]
## Overview 
This ticket is to implement the front-end changes as shown in the updated design https://github.com/department-of-veterans-affairs/va.gov-team/issues/33772

## Tasks
- [x] Build FE to display modals as shown in the latest mockups
- [x] Test
- [ ] Leave comment in this ticket when ready to review on Staging

## Acceptance Criteria
- [ ] Build FE to display modals as shown in the latest mockups
- [ ] Test
- [ ] Leave comment in this ticket when ready to review on Staging

**Comment by adamwhitlock1** [comment]
This should be ready to test with this test user vets.gov.user+75@gmail.com in staging. 

I am still seeing the redirect error that was resolved in prod earlier in staging, but once that clears up we should be able to get eyes on this.

**Comment by MarciMcGuireGCIO** [comment]
Thanks @adamwhitlock1 ! I’ll review as soon as that is resolved (or after grooming, whichever happens first)

**Comment by MarciMcGuireGCIO** [comment]
Hey @adamwhitlock1.  I was able to log in with user 75, but I'm seeing something weird.  There's a button labeled "Trigger Mailing Address Failure", and when I click Yes that I want to update my mailing address, it takes me back to the page still in Edit state, and when I try to update, this modal displays again.
![image](https://user-images.githubusercontent.com/73354907/158659071-bf0c50e9-f615-4900-aed7-0260ec0cbb39.png)

**Comment by MarciMcGuireGCIO** [comment]
@Samara-Strauss - if you can review on Staging and provide any feedback, that will be great.  Thanks!

**Comment by Samara-Strauss** [comment]
A couple things:

- For some reason, "DC" and the zipcode are not showing when they are included in the address I saved.
- Idk if this just isn't built yet, but we need a better state for when there is no mailing address then to just show a blank space.

![image](https://user-images.githubusercontent.com/34068740/159362125-8641a6cc-bcc1-4129-88d2-2f829d0ae036.png)

**Comment by adamwhitlock1** [comment]
hmmm that is strange. Maybe try once again @Samara-Strauss I was able to see it correctly on my end, but I wonder if there is something flakey about the state and zip code display since this isnt wired to the api call.

We do have a ticket here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/37842 that allows the content to be conditional to deal with a mailing address missing, so that it isnt blank in the UI display.

**Comment by Samara-Strauss** [comment]
@adamwhitlock1 it doesn't look like I can test that use case again because a mailing address was added to user 75, and we don't allow people to delete mailing addresses for me to duplicate the same scenario. However, the issue of the state/zip code not showing up did not happen again when I looked at it again. Idk if this is a weird bug that only occurs when there is not a mailing address (not sure why that would happen), but we can validate the no-mailing-address use case when this functionality is available to more staging users.
